aspose/src/main/java/nl/nn/adapterframework/extensions/aspose/services/conv/CisConversionResult.java

```diff
@@ -205,13 +205,12 @@ public final String toString() {
 	 */
 	public void buildXmlFromResult(XmlBuilder main, CisConversionResult cisConversionResult, boolean isRoot) {
 		if(isRoot) {
-			main.addAttribute("CONVERSION_OPTION", this.getConversionOption().getValue());
-			main.addAttribute("MEDIA_TYPE", this.getMediaType().toString());
-			main.addAttribute("DOCUMENT_NAME", this.getDocumentName());
-			main.addAttribute("FAILURE_REASON", this.getFailureReason());
-			main.addAttribute("PARENT_CONVERSION_ID", null);
-			main.addAttribute("NUMBER_OF_PAGES", this.getNumberOfPages());
-			main.addAttribute("CONVERTED_DOCUMENT", this.getResultFilePath());
+			main.addAttribute("conversionOption", this.getConversionOption().getValue());
+			main.addAttribute("mediaType", this.getMediaType().toString());
+			main.addAttribute("documentName", this.getDocumentName());
+			main.addAttribute("failureReason", this.getFailureReason());
+			main.addAttribute("numberOfPages", this.getNumberOfPages());
+			main.addAttribute("convertedDocument", this.getResultFilePath());
```

Comment:
Het is wel handig om hier unit tests voor te hebben. Blijkbaar kunnen deze dingen zomaar veranderen??

Reply:
Ik ben het zeker eens met het feit dat unit tests handig zijn alleen weet ik niet wanneer ik deze kan toevoegen.

Daarnaast was het geval dat we vandaag via Skype hadden besproken omdat this.getResultFilePath() een null waarde terug gaf waardoor dit attribuut niet werd toegevoegd aan de XML.